keyboards/soda/mango/info.json

```diff
@@ -0,0 +1,106 @@
+{
+    "keyboard_name": "SodaMango",
+    "url": "",
+    "maintainer": "qmk",
+    "manufacturer": "JeayoKeh",
+    "usb": {
+        "vid": "0x1A54",
```

Comment:
This VID is in use, and should be changed.
https://www.the-sz.com/products/usbid/index.php?v=0x1A54&p=&n=

Generally, above 0x4000 isn't assigned yet, and the further past, the better.